### ISSUE TITLE
Fix request: false positive in BufferFilesNewStream 

### ISSUE THREAD

**Issue by jborgers** [jborgers/PMD-jPinpoint-rules#273]
```java
try (InputStream is1 = Files.newInputStream(file.toPath())) { //NOPMD //NOSONAR - suppressed BufferFilesNewStream - IOUtils.readLines already buffers
    System.out.print(String.format("INFO: processing '%s'", file.getName()));
    List<String> file1Lines = IOUtils.readLines(is1, Charset.defaultCharset()); // uses a BufferedReader
```

**Comment by jborgers** [comment]
IOUtils javadoc: 
`All the methods in this class that read a stream are buffered internally. This means that there is no cause to use a BufferedInputStream or BufferedReader. `
